source/core/serial_solid.cc

```diff
@@ -589,13 +561,16 @@ void SerialSolid<dim, spacedim>::read_checkpoint(std::string /*prefix*/)
 
 template <int dim, int spacedim>
 void
-SerialSolid<dim, spacedim>::setup_displacement()
+SerialSolid<dim, spacedim>::setup_dof_handler()
 {
   displacement_dh.distribute_dofs(*this->displacement_fe);
 
   displacement.reinit(displacement_dh.n_locally_owned_dofs());
+  displacement_since_mapped.reinit(displacement_dh.n_locally_owned_dofs());
+
```

Comment:
Why all those extra lines :(? Is there a purpose? Because there are some after this comment, but I will stop x)

Reply:
Not sure really. I coded this over multiple weeks so things might have drifted.